src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EEMethodSymbol.cs

```diff
@@ -436,6 +436,8 @@ internal override ObsoleteAttributeData ObsoleteAttributeData
 
         internal override bool HasUnscopedRefAttribute => false;
 
+        internal override bool UseUpdatedEscapeRules => false;
```

Comment:
If we never expect to read this property, because this method is not callable in user code, it might be reasonable to `throw ExceptionUtilities.Unreachable` here.